packages/esm-patient-search-app/src/mpi/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { capitalize } from 'lodash-es';
+import { type SearchedPatient } from '../types';
+import { getCoreTranslation } from '@openmrs/esm-framework';
+export function inferModeFromSearchParams(searchParams: URLSearchParams) {
+  return searchParams.get('mode')?.toLowerCase() === 'external' ? 'external' : 'internal';
+}
+
+export function mapToOpenMRSPatient(fhirPatients: Array<any>): Array<SearchedPatient> {
```

Comment:
This should have a stronger type than `any`. We are definitely assuming that certain properties exist and that should be reflected in the type here.